src/routes/owners/owners.service.ts

```diff
@@ -0,0 +1,19 @@
+import { Inject, Injectable } from '@nestjs/common';
+import { SafeList } from '../../domain/safe/entities/safe-list.entity';
+import { SafeRepository } from '../../domain/safe/safe.repository';
+import { ISafeRepository } from '../../domain/safe/safe.repository.interface';
+
+@Injectable()
+export class OwnersService {
+  constructor(
+    @Inject(ISafeRepository)
+    private readonly safeRepository: SafeRepository,
+  ) {}
+
+  async getSafesByOwner(
+    chainId: string,
+    ownerAddress: string,
+  ): Promise<SafeList> {
```

Comment:
What is the reason to not use the route entity at the route level?

Reply:
No reason, just a bad import, thanks! 🙂

Fixed in [e425213](https://github.com/5afe/safe-client-gateway-nest/pull/192/commits/e425213378dbb5b522c226e9d662b5432df1cc92)